tests/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# cmake-format: off
+# tests/CMakeLists.txt -*-makefile-*-
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# cmake-format: on
+
+include(GoogleTest)
+
+add_executable(beman.iterator_interface.tests)
+
+target_sources(beman.iterator_interface.tests
+               PRIVATE iterator_interface.test.cpp)
+
+target_sources(
+  beman.iterator_interface.tests
+  PRIVATE FILE_SET beman_iterator_interface_tests_headers TYPE HEADERS)
+
+target_link_libraries(
+  beman.iterator_interface.tests PRIVATE beman::iterator_interface GTest::gtest
+                                         GTest::gtest_main)
+
+# Issue #32: Re-enable ASAN run CI/clang-19.
```

Comment:
```suggestion
# Issue #32: Re-enable ASAN run CI/clang-19.
```

I won't post a number here. If we perform a copy again, we may not have the same issue number